lib/spack/spack/cmd/extensions.py

```diff
@@ -25,12 +24,12 @@ def setup_parser(subparser):
     format_group.add_argument(
```

Comment:
This mutually exclusive format group is no longer needed if `--long` is the only thing in it.

Reply:
fixed.